src/computeFontSize.ts

```diff
@@ -0,0 +1,63 @@
+import * as vscode from "vscode";
+
+export interface FontSize {
+  fontSize: number;
+  fontWidth: number;
+  fontHeight: number;
+}
+
+export default async function (): Promise<FontSize> {
```

Comment:
wow i didn't know you could define an anonymous function for export this way

Reply:
Neither did I. I just trying what I think TS should do :D